src/main/scala/akka/persistence/eventstore/query/scaladsl/EventStoreReadJournal.scala

```diff
@@ -63,6 +75,25 @@ class EventStoreReadJournal(system: ExtendedActorSystem, config: Config)
       if (to > Int.MaxValue) EventNumber.Last else eventNumber(to))
   }
 
+  private def eventsByTag(tag: String, from: Long, infinite: Boolean): Source[EventEnvelope, akka.NotUsed] = {
+    val streamId = EventStream.System(s"ce-$tag")
```

Comment:
Exactly. The reason and the limitations are obvious :-) I guess there should be no restriction on the **type** (stream ID, stream category) of the input tag. The plugin should simply detect the type.